### ISSUE TITLE
behavior of combine() on the end of the time series

### ISSUE THREAD

**Issue by maxlarsen** [JuliaStats/TimeSeries.jl#245]
Although the code is careful (cf comments in the code) to take care of what happens for the last values which might not constitute a complete week (for instance), I don't think it implements the expected behavior: `lasttempvalue = f(ta.values)` means that if we have Monday-Monday=1:8 and we `combine(f=first)`, the second week will collapse to 1 instead of 8. And indeed:

~~~
julia> ts=TimeArray(collect(Date(2012,1,2):Date(2012,1,9)),collect(1:8),["ts"])
8x1 TimeSeries.TimeArray{Int64,1,Date,Array{Int64,1}} 2012-01-02 to 2012-01-09

             ts      
2012-01-02 | 1       
2012-01-03 | 2       
2012-01-04 | 3       
2012-01-05 | 4       
⋮
2012-01-06 | 5       
2012-01-07 | 6       
2012-01-08 | 7       
2012-01-09 | 8       


julia> collapse(ts,first,period=week)
2x1 TimeSeries.TimeArray{Float64,1,Date,Array{Float64,1}} 2012-01-08 to 2012-01-09

             ts      
2012-01-08 | 1       
2012-01-09 | 1   
~~~

**Comment by GordStephen** [comment]
I think this is fixed in an in-progress PR (#244):

```julia
julia> collapse(ts,first,period=Dates.week)
2x1 TimeSeries.TimeArray{Int64,1,Date,Array{Int64,1}} 2012-01-08 to 2012-01-09

             ts      
2012-01-08 | 1       
2012-01-09 | 8
```

The updated code is in the `collapse` branch, if you want to try it out.

**Comment by maxlarsen** [comment]
Thank you.

**Comment by milktrader** [comment]
Closed by #244. 

The correct method call for this example is

````julia
 julia> collapse(ts,week, first)

2x1 TimeSeries.TimeArray{Int64,1,Date,Array{Int64,1}} 2012-01-02 to 2012-01-09

             ts      
2012-01-02 | 1       
2012-01-09 | 8       
````
